geoips/plugins/modules/output_checkers/text.py

```diff
@@ -23,6 +23,60 @@
 name = "text"
 
 
+def clear_text(match_path, close_path, bad_path):
+    """Clear output text files so they can be written again."""
+    open(match_path, "w").close()
+    open(close_path, "w").close()
+    open(bad_path, "w").close()
+
+
+def get_test_files():
+    """Return a series of varied text files."""
+    import numpy as np
+    from shutil import copy
+    from os import mkdir, environ
+    from os.path import exists
+
+    savedir = str(environ["GEOIPS_OUTDIRS"]) + "/scratch/unit_tests/test_text/"
+    if not exists(savedir):
+        mkdir(savedir)
+    comp_path = savedir + "compare.txt"
+    match_path = savedir + "matched.txt"
+    close_path = savedir + "close_mismatch.txt"
+    bad_path = savedir + "bad_mismatch.txt"
+    clear_text(match_path, close_path, bad_path)
+    copy(comp_path, match_path)
```

Comment:
Where is "comp_path" coming from?  I get file does not exist when attempting to run this - should it be auto-generated in the "savedir" directly within this method before attempting to copy/modify it?

Reply:
Ahh I forgot about this. I'll have to change this for fdeck, text, and geotiff. Should be pretty easy though. Only problem that might occur is for fdeck, since I copied that file from the web.

Reply:
Working now!
==================================== 1413 passed, 2 warnings in 23.60s ====================================